test-cases/features/elasticity/scalein-scaleout-elasticity-90-percent.yaml

```diff
@@ -0,0 +1,42 @@
+test_duration: 3000
+prepare_write_cmd: [
+  "cassandra-stress write no-warmup cl=ALL n=300000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..300000000",
+  "cassandra-stress write no-warmup cl=ALL n=300000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=300000001..600000000",
+  "cassandra-stress write no-warmup cl=ALL n=300000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=600000001..900000000",
+  "cassandra-stress write no-warmup cl=ALL n=250000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=900000001..1200000000",
+  "cassandra-stress write no-warmup cl=ALL n=300000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1200000001..1500000000",
+]
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=ALL n=335544320 -schema 'keyspace=refill_keyspace replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=300 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1500000001..1835544321"
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=800m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate 'threads=300 fixed=16875/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..600000000,300000000,6000000)'"
+
+simulated_racks: 3
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+
+instance_type_loader: 'c6i.2xlarge'
```

Comment:
c7i should be better.
@Lakshmipathi did you verify how much cpu is utilized on loaders? maybe we could reduce the size?

Reply:
Loader nodes utilization:
![Screenshot from 2025-05-09 17-15-30](https://github.com/user-attachments/assets/38fb6899-f23c-4ede-91ac-3fa692e472cf)

Source: [scylla-staging/LakshmipathiGanapathi/scylla-enterprise-perf-regression-latency-650gb-elasticity#98](https://argus.scylladb.com/tests/scylla-cluster-tests/8a90107a-8c81-4332-a264-767f47351a26)

RAM-wise loader nodes have big margin, but CPU-wise it exceeds 50% utilization even without 5->4 disbalance happening in the pre-load step.

So, not changing summary load values, it may stay as-is, I think.

Reply:
ok, in that case we cannot reduce this size. Unless this strange load we pose to Scylla is bogus... - @Lakshmipathi what is the reason of this higher load?